packages/sdk/client/src/echo/agent-query-source-provider.ts

```diff
@@ -16,6 +16,8 @@ import { type EchoObject as EchoObjectProto } from '@dxos/protocols/proto/dxos/e
 import { type QueryRequest, type QueryResponse } from '@dxos/protocols/proto/dxos/echo/query';
 import { type GossipMessage } from '@dxos/protocols/proto/dxos/mesh/teleport/gossip';
 
+const ERR_CLOSING = new Error();
```

Comment:
no message?

Reply:
it's just a "marker" exception to check by value if we threw this.
@mykola-vrmchk i think this is better than checking the "text" value.